stable/artifactory-ha/templates/artifactory-service.yaml

```diff
@@ -49,7 +49,9 @@ spec:
   {{- end }}
   {{- end }}
   selector:
-{{- if eq .Values.artifactory.service.pool "members" }}
+{{- if eq (int .Values.artifactory.node.replicaCount) 0 }}
```

Comment:
Same here. If no error, leave this for manual scaling.

Reply:
As discussed , following with the same approach.